Cheeze buttons (normal buttons) give too many lua errors when trying to place them

When placing one of the wiremod buttons using normal button tool, errors occur and I get too many lua errors:

``` lua
TEST [lua/vgui/dnumslider_legacy.lua][lua/vgui/propselect.lua]
TEST [lua/vgui/dnumslider_legacy.lua][lua/vgui/dlabel.lua]

[ERROR] lua/vgui/propselect.lua:54: bad argument #1 to 'pairs' (table expected, got boolean)
  1. pairs - [C]:-1
   2. DoClick - lua/vgui/propselect.lua:54
    3. unknown - lua/vgui/dlabel.lua:206

Disconnect: "Too many Lua Errors! Sorry!".
Disconnect: "Too many Lua Errors! Sorry!".
```

--- a/lua/wire/client/cl_modelplug.lua
+++ b/lua/wire/client/cl_modelplug.lua
@@ -131,7 +131,7 @@ local CheezesButtons = {
 }
 for k,v in ipairs(CheezesButtons) do
 	if file.Exists(v,"GAME") then
-		list.Set( "ButtonModels", v, true )
+		list.Set( "ButtonModels", v, {} )
 		list.Set( "Wire_button_Models", v, true )
 	end
 end
@@ -149,7 +149,7 @@ local CheezesSmallButtons = {
 }
 for k,v in ipairs(CheezesSmallButtons) do
 	if file.Exists(v,"GAME") then
-		list.Set( "ButtonModels", v, true )
+		list.Set( "ButtonModels", v, {} )
 		list.Set( "Wire_button_small_Models", v, true )
 	end
 end

